tests/relay/test_fields.py

```diff
@@ -581,6 +581,35 @@ async def test_query_connection_filtering_first_async(mocker, query_attr: str):
     }
 
 
+def test_query_connection_filtering_after_without_first():
+    result = schema.execute_sync(
+        """{ someFruits {
+            edges { node { id } }
+            pageInfo {
+                endCursor
+            }
+        } }"""
+    )
+    assert not result.errors
+    assert len(result.data["someFruits"]["edges"]) == 100
+    assert (
+        relay.from_base64(result.data["someFruits"]["edges"][99]["node"]["id"])[1]
+        == "99"
+    )
+    result = schema.execute_sync(
+        """{ someFruits(after: "%s", first: 100) {
+            edges { node { id } }
+        } }"""
+        % result.data["someFruits"]["pageInfo"]["endCursor"]
+    )  #
```

Comment:
this is a nit pick, but could you use variables here instead of string formatting? 😊

Reply:
done